models/Capacity.v1.yaml

```diff
@@ -0,0 +1,21 @@
+title: Capacity
+type: object
+x-examples:
+  default:
+    timestamp: '2020-07-16T17:05:08-07:00'
+    timezone: America/Vancouver
+    hours_to_calculate: 8
+description: "The root of the Capacity's schema."
+properties:
+  timestamp:
+    type: string
+    description: 'Current local time with offset Eg: "2020-07-16T17:05:08-07:00"'
```

Comment:
As far as I can see, this model is only used in a request body. I'm not sure what kind of math we're hoping to do on the backend with that. What if the user sends a timestamp like this:

`2222-07-16T17:05:08-07:00`

Will we assume that the local time is 202 years in the future?

Reply:
That is an interesting question. 

So you mean we never pass the current_time, because we assume it will not be correct always.

But will there be a scenario where they can pass a time and we compute the capacity for that time and the expected invites for the next number of hours?

Reply:
In that case, we aren't asking the CURRENT time, but the time at which would be the starting point for the capacity calculations. In that case, if the user asked for capacity management starting in the year 2222, we'd be able to use that as the starting point - as opposed to assuming that the location fell into some kind of time warp :D